src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/IRfc3161TimestampTokenInfo.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Security.Cryptography;
+
+namespace NuGet.Packaging.Signing
+{
+    internal interface IRfc3161TimestampTokenInfo
+    {
+#if IS_SIGNING_SUPPORTED
```

Comment:
very minor considering the type is `internal`, but why not put the `#if` outside the interface definition, so the interface only exists on TFMs that support signing?

Reply:
Thanks. Will fix in #3571